cluster/aws/templates/common.sh

```diff
@@ -15,11 +15,15 @@
 # limitations under the License.
 
 
-apt-get update
-apt-get install --yes curl
+until apt-get update; do
+  echo "== apt-get update failed, retrying =="
+  echo sleep 5
+done
+
+apt-get install -y curl
+apt-get dist-upgrade -y
```

Comment:
We don't control what versions are in the base vivid image anyway. I think although you're changing the safe environment that you've tested many times over, this can't really conceivably break anything, and the security updates are important enough to take that risk.